lib/salus.rb

```diff
@@ -55,6 +58,19 @@ def scan(
                                        ignore_config_id: ignore_config_id,
                                        cli_scanners_to_run: only)
 
+      if sarif_diff_full != ""
```

Comment:
Minor nit... can we break out this logic to a method to keep the current scan method from growing too much.

`process_sarif_full_diff(sarif_diff_full) unless sarif_diff_full.empty?`

      
```

Reply:
updated accordingly.